### ISSUE TITLE
Init error 0x13 in one Daytona machine but not in the other one with the different BMC version.

### ISSUE THREAD

**Issue by MelodyHuibo** [AMDESE/AMDSEV#85]
Hi, I have two Daytona machines， BIOS in both machines is RYM1008B， one machine with BMC 03.55.37.0008 can enable SEV-SNP， the other one with BMC03.50.22.0008 could not enable SEV-SNP(0x13 error), Is that possible that the 0x13 error is related to the BMC version?? Any idea?

**Comment by lilienbm** [comment]
Hi,

No. It's not related to the BMC. SNP giving 0x13 most likely means your
iommu settings are not correct or your bios settings are incorrect. Do you
have the same kernel installed on both machines and for sure have set
SNP_SUPPORT=Enabled on both?

**Comment by MelodyHuibo** [comment]
Yes I do have the same kernel and same bios settings and firmware, So try
to figure out the possible root cause since I have no clue now

**Comment by lilienbm** [comment]
What happens when you put the working hard drive into the other platform?

**Comment by MelodyHuibo** [comment]
Have no tried it yet. Let us try it and see what happens
